internal/database/migrations.go

```diff
@@ -440,71 +443,77 @@ var migrations = []func(tx *sql.Tx) error{
 			return err
 		}
 
-		_, err = tx.Exec(`
-			DECLARE my_cursor CURSOR FOR
-			SELECT
-				id,
-				COALESCE(extra->'custom_css', '') as custom_css,
-				COALESCE(extra->'google_id', '') as google_id,
-				COALESCE(extra->'oidc_id', '') as oidc_id
-			FROM users
-			FOR UPDATE
-		`)
-		if err != nil {
-			return err
-		}
-		defer tx.Exec("CLOSE my_cursor")
+		if driver == "postgresql" {
+			_, err = tx.Exec(`
+				DECLARE my_cursor CURSOR FOR
+				SELECT
+					id,
+					COALESCE(extra->'custom_css', '') as custom_css,
+					COALESCE(extra->'google_id', '') as google_id,
+					COALESCE(extra->'oidc_id', '') as oidc_id
+				FROM users
+				FOR UPDATE
+			`)
+			if err != nil {
+				return err
+			}
+			defer tx.Exec("CLOSE my_cursor")
 
-		for {
-			var (
-				userID           int64
-				customStylesheet string
-				googleID         string
-				oidcID           string
-			)
+			for {
+				var (
+					userID           int64
+					customStylesheet string
+					googleID         string
+					oidcID           string
+				)
 
-			if err := tx.QueryRow(`FETCH NEXT FROM my_cursor`).Scan(&userID, &customStylesheet, &googleID, &oidcID); err != nil {
-				if err == sql.ErrNoRows {
-					break
+				if err := tx.QueryRow(`FETCH NEXT FROM my_cursor`).Scan(&userID, &customStylesheet, &googleID, &oidcID); err != nil {
+					if err == sql.ErrNoRows {
+						break
+					}
+					return err
 				}
-				return err
-			}
 
-			_, err := tx.Exec(
-				`UPDATE
-					users
-				SET
-					stylesheet=$2,
-					google_id=$3,
-					openid_connect_id=$4
-				WHERE
-					id=$1
-				`,
-				userID, customStylesheet, googleID, oidcID)
-			if err != nil {
-				return err
+				_, err := tx.Exec(
+					`UPDATE
+						users
+					SET
+						stylesheet=$2,
+						google_id=$3,
+						openid_connect_id=$4
+					WHERE
+						id=$1
+					`,
+					userID, customStylesheet, googleID, oidcID)
+				if err != nil {
+					return err
+				}
 			}
 		}
 
 		return err
 	},
-	func(tx *sql.Tx) (err error) {
+	func(tx *sql.Tx, driver string) (err error) {
+		if driver == "postgresql" {
+			if _, err = tx.Exec(`ALTER TABLE users DROP COLUMN extra;`); err != nil {
+				return nil
```

Comment:
Shouldn't this be `return err` like in the other cases?

Reply:
You're absolutely right: https://github.com/miniflux/v2/pull/3049